libraries/USBHost/src/hidboot.h

```diff
@@ -312,8 +312,8 @@ uint32_t HIDBoot<BOOT_PROTOCOL>::Init(uint32_t parent, uint32_t port, uint32_t l
 
 
 	// Reset
-	UHD_BusReset();
-	while( Is_uhd_starting_reset() ) {}
+	//UHD_BusReset();
+	//while( Is_uhd_starting_reset() ) {}
```

Comment:
@gdsports @facchinm is we don't need these lines any more, let's remove instead of commenting out